pkgs/by-name/fi/firewalk/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchurl
+, libnet
+, libpcap
+, libdnet
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firewalk";
+  version = "5.0";
+
+  src = fetchurl {
+    url = "http://web.archive.org/web/20060925223218/http://www.packetfactory.net/firewalk/dist/firewalk-5.0.tgz";
```

Comment:
Have you verified that this builds? Page says 2003, if this works it's a miracle